nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -74,7 +75,8 @@ def quantize_with_accuracy_control(model: ov.Model,
                                    subset_size: int = 300,
                                    fast_bias_correction: bool = True,
                                    model_type: Optional[ModelType] = None,
-                                   ignored_scope: Optional[IgnoredScope] = None) -> ov.Model:
+                                   ignored_scope: Optional[IgnoredScope] = None,
+                                   algo_backend: Optional[OVAccuracyControlAlgoBackend] = None) -> ov.Model:
```

Comment:
These changes are needed because we should not compile the OV model when we pass it to the model evaluator.
So the `algo_backend.prepare_for_inference()` method should just return the passed model.

We may remove this parameter. But in this case, we should create another method to use the different backend.

Reply:
Mmm... Creating OVNativeAccuracyControlAlgoBackend does not solve the issue?

Reply:
Not applicable now. The code was changed.